loader/src/smart-scheduling-links.js

```diff
@@ -38,11 +38,16 @@ const EXTENSIONS = Object.freeze({
 const PRODUCTS_BY_CVX_CODE = {
   210: "astra_zeneca",
   207: "moderna",
+  221: "moderna",
   211: "novavax",
   208: "pfizer",
+  217: "pfizer",
   218: "pfizer_age_5_11",
   219: "pfizer_age_2_4",
```

Comment:
Any reason not to go ahead and rename this `pfizer_age_0_4`?

Reply:
Yep — NJ already knows about and is using it. I’d like to rename, but want a go-ahead from them first: #710.

Reply:
Got a 👍 there so will also tweak that name before merging.